doc/articles/features/windows-ui-succinct-syntax.md

```diff
@@ -0,0 +1,40 @@
+# Uno Support for WinUI succinct syntax
```

Comment:
Very nice! I think we can add this documentation as a Grid control entry directly (at the same level as articles/controls/ComboBox.md).



```suggestion
# The Grid panel
Defines a flexible grid area that consists of columns and rows. See the [Microsoft docs](https://docs.microsoft.com/en-us/windows/winui/api/microsoft.ui.xaml.controls.grid?view=winui-3.0) for more details.

## Uno Support for WinUI succinct syntax
```

Reply:
I added documentation to features because it's not a Grid specific feature. This syntax also extends to many other controls, such as `<ListView Items="'Hello', 'world'" />`: https://github.com/microsoft/microsoft-ui-xaml-specs/blob/master/ active/gridsyntax/GridSyntaxSpec.md#other-use-cases-for-succinct-syntax. Of course, at the moment we have only implemented Grid, but I noted this in the documentation

Reply:
I see, it makes sense for future additions when we do support other controls. Thanks!